ui/app/components/app/user-preferenced-currency-display/user-preferenced-currency-display.container.js

```diff
@@ -1,67 +0,0 @@
-import { connect } from 'react-redux'
-import UserPreferencedCurrencyDisplay from './user-preferenced-currency-display.component'
-import { preferencesSelector, getIsMainnet } from '../../../selectors'
-import { ETH, PRIMARY, SECONDARY } from '../../../helpers/constants/common'
-
-const mapStateToProps = (state) => {
-  const {
-    useNativeCurrencyAsPrimaryCurrency,
-    showFiatInTestnets,
-  } = preferencesSelector(state)
-
-  const isMainnet = getIsMainnet(state)
-
-  return {
-    useNativeCurrencyAsPrimaryCurrency,
-    showFiatInTestnets,
-    isMainnet,
-    nativeCurrency: state.metamask.nativeCurrency,
-  }
-}
-
-const mergeProps = (stateProps, dispatchProps, ownProps) => {
-  const { useNativeCurrencyAsPrimaryCurrency, showFiatInTestnets, isMainnet, nativeCurrency, ...restStateProps } = stateProps
-  const {
-    type,
-    numberOfDecimals: propsNumberOfDecimals,
-    ethNumberOfDecimals,
-    fiatNumberOfDecimals,
-    ethPrefix,
-    fiatPrefix,
-    prefix: propsPrefix,
-    ...restOwnProps
-  } = ownProps
-
-  let currency, numberOfDecimals, prefix
-
-  if ((type === PRIMARY && useNativeCurrencyAsPrimaryCurrency) ||
-    (type === SECONDARY && !useNativeCurrencyAsPrimaryCurrency)) {
-    // Display ETH
-    currency = nativeCurrency || ETH
-    numberOfDecimals = propsNumberOfDecimals || ethNumberOfDecimals || 6
-    prefix = propsPrefix || ethPrefix
-  } else if ((type === SECONDARY && useNativeCurrencyAsPrimaryCurrency) ||
-    (type === PRIMARY && !useNativeCurrencyAsPrimaryCurrency)) {
-    // Display Fiat
-    numberOfDecimals = propsNumberOfDecimals || fiatNumberOfDecimals || 2
-    prefix = propsPrefix || fiatPrefix
-  }
-
-  if (!isMainnet && !showFiatInTestnets) {
-    currency = nativeCurrency || ETH
-    numberOfDecimals = propsNumberOfDecimals || ethNumberOfDecimals || 6
-    prefix = propsPrefix || ethPrefix
```

Comment:
It looks like this `prefix` variable isn't being set by your replacement hooks, so the `ethPrefix` and `fiatPrefix` props are now ignored.

Though I'm not either of these props are used anywhere 🤔 Perhaps they can be removed instead? I noticed they're still present in the propTypes.

Reply:
I opted to remove them from propTypes. I had come to the same conclusion regarding no longer being used, but didn't finish the job of removing them 😅